pkg/models/forum/comment.go

```diff
@@ -75,21 +88,39 @@ func (f *ForumStore) RateComment(commentID, userID int) (bool, error) {
 	return rateOk, err
 }
 
-func (f *ForumStore) GetCommentRatings(commentID int) (likes []int64, err error) {
+func (f *ForumStore) getCommentLikers(commentID int) (likersIDs []int64, err error) {
 	rows, err := f.DB.Query(`SELECT user_id FROM ratings WHERE comment_id = $1;`, commentID)
 	if err != nil {
```

Comment:
Where is return statement? If sql query fails, application will throw error caused by referencing to invalid memory address or nil pointer dereference.